src/main/studio/ui/SearchPanel.java

```diff
@@ -160,22 +162,37 @@ private void doSearch(SearchContext context, SearchAction action) {
                     result = SearchEngine.replaceAll(textArea, context);
                 }
             } catch (IndexOutOfBoundsException e) {
-                editorPane.setTemporaryStatus("Error during replacement: " + e.getMessage());
+                editorPane.setTemporaryStatus("Error during replacement: " + e.getMessage(), true);
```

Comment:
(not related to this line)... Actually replace doesn't work correctly. If you start replacing text (just clicking replace button), every other found entry would be skipped. I believe that to fix this the line 151 above needs to be replaced to something like the following:
```
        boolean searchFromSelStart;
        if (context.getReplaceWith() != null) {
            searchFromSelStart = context.getSearchForward();
        } else {
            searchFromSelStart = ! context.getSearchForward();
        }
        int pos = searchFromSelStart ? textArea.getSelectionStart() : textArea.getSelectionEnd();
```

I propose you to add this change (I wanted to do it myself, but unfortunately didn't have time :( )

Reply:
I'll put this into a separate task:
#108